### DISCUSSION TITLE
Does Appium 2.0 supports in AWS?

### DISCUSSION THREAD

**Discussion by deepu451** [appium/appium#17587] [Q&A]
Hello Everyone,
can anyone confirm Appium 2.0 supports in AWS?

**Comment by KazuCocoa** [comment]
You may need to contact the support team

**Comment by jlipps** [comment]
Do you mean does AWS DeviceFarm support Appium 2.0? That's not a question I know the answer to; contact AWS support. I'm guessing the answer may be no, currently.

**Reply by deepu451** [reply]
yes, i mean device farm only. Thank you. I will reach AWS support team.
